vowpalwabbit/ftrl.cc

```diff
@@ -168,7 +168,7 @@ void inner_update_pistol_post(update_data& d, float x, float& wref)
 // W_MX 3  maximum absolute value
 // W_WE 4  Wealth
 // W_MG 5  Maximum Lipschitz constant
-void inner_update_cb_state_and_predict(update_data& d, float x, float& wref)
+void inner_coin_betting_predict(update_data& d, float x, float& wref)
```

Comment:
was dropping the 'update' word an intended thing?

Reply:
Yes.